### PR TITLE
dpazykovIO

### PR DESCRIPTION
**Pull request by Daniel12122**

### CODE REVIEW

File: src/main/java/IO/Main.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class Main {
+
+    private static final String PATH = System.getProperty("user.dir") + File.separator
+            + "src" + File.separator + "main" + File.separator + "resources" + File.separator;
+
+    private static String fileNameCSV = PATH + "CSVFile.csv";
+    private static String fileNameXLS = PATH + "XLSFile.xls";
+
+    public static void main(String[] args) throws IOException {
```
Comment (AqaCoreSchool) [inline src/main/java/IO/Main.java]: not a best approach, commented in telegram group

File: src/main/java/IO/IO.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class IO {
+
+    private Faker faker = new Faker();
+
+    private void exists(String fileName) throws IOException {
```
Comment (AqaCoreSchool) [inline src/main/java/IO/IO.java]: poor naming, I would expect return type of boolean from method with name "exists"
this method looks like redundant and can be avoided by 
`if (file = null) {
file.createNewFile()
}`

File: src/main/java/IO/IO.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            }
+        }
+    }
+
+    public List<String> readFileToList(String fileName) throws IOException {
+        File file = new File(fileName);
+        List<String> dataList = new ArrayList<>();
+        exists(fileName);
+        try (BufferedReader in = new BufferedReader(new FileReader(file.getAbsoluteFile()))) {
+                while (true) {
```
Comment (AqaCoreSchool) [inline src/main/java/IO/IO.java]: you should avoid using while(true) cause there is no exit condition and you can get into infinite loop
could used while(in.readLine() == null)

File: src/main/java/IO/IO.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        }
+        return dataList;
+    }
+
+    public void delete(String nameFile) throws IOException {
+        exists(nameFile);
+        new File(nameFile).delete();
+    }
+
+    public void writeExcelFile(List<String> inputDataList,String fileName) throws IOException {
```
Comment (AqaCoreSchool) [inline src/main/java/IO/IO.java]: why throws IOexc and handles it at same time?

File: src/main/java/IO/IO.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        HSSFWorkbook hssfWorkbook = new HSSFWorkbook();
+        HSSFSheet hssfSheet = hssfWorkbook.createSheet("XLSFile");
+        int bound = inputDataList.size();
+        IntStream.range(0, bound).forEach(i -> {
+            HSSFRow hssfSheetRow = hssfSheet.createRow(i);
+            HSSFCell hssfCell = hssfSheetRow.createCell(i);
+            hssfCell.setCellValue(inputDataList.get(i));
+        });
+        try{
+            FileOutputStream output = new FileOutputStream(fileName);
```
Comment (AqaCoreSchool) [inline src/main/java/IO/IO.java:77]: what about closing this stream?

File: src/main/java/IO/Main.java (hunk 1 above)
Comment (Daniel12122) [reply src/main/java/IO/Main.java]: Fix

File: src/main/java/IO/IO.java (hunk 2 above)
Comment (Daniel12122) [reply src/main/java/IO/IO.java]: fix

File: src/main/java/IO/IO.java (hunk 4 above)
Comment (Daniel12122) [reply src/main/java/IO/IO.java]: fix

File: src/main/java/IO/IO.java (hunk 5 above)
Comment (Daniel12122) [reply src/main/java/IO/IO.java:77]: fix

File: src/main/java/IO/IO.java (hunk 3 above)
Comment (Daniel12122) [reply src/main/java/IO/IO.java]: fix

**Pull request merged**